templates/corporate/support/next_plan_forms_support.html

```diff
@@ -1,8 +1,21 @@
 <h4>⏱️ Schedule fixed price plan:</h4>
+{% set current_plan_is_billable = (
+  current_plan and
+  current_plan.tier in [
+  current_plan.TIER_SELF_HOSTED_BASIC,
+  current_plan.TIER_SELF_HOSTED_BUSINESS
+  ]
+  ) %}
```

Comment:
We shouldn't have a list of plans hardcoded in template files; we should pass the determination as to whether the current plan is billable in from the caller. And probably that code should call `check_plan_tier_is_billable`.

Reply:
Made the changes.